scheduler/src/cook/mesos/schema.clj

```diff
@@ -117,6 +117,16 @@
     :db/valueType :db.type/ref
     :db/cardinality :db.cardinality/one
     :db.install/_attribute :db.part/db}
+   {:db/id (d/tempid :db.part/db)
+    :db/ident :job/last-fenzo-placement-failure
+    :db/valueType :db.type/string
```

Comment:
Can you add `:db/doc`s to these? Want to start getting better at documenting the schema (and everything else)